extras/python/fogbench/__main__.py

```diff
@@ -213,6 +210,9 @@ def read_out_file(_file=None, _keep=False, _iterations=1, _interval=0, send_to='
      URI "/other/sensor-values".
 
     """
+    from aiocoap import Context, Message, numbers
+    from cbor2 import dumps
+
     context = await Context.create_client_context()
 
     request = Message(payload=dumps(payload), code=POST)
```

Comment:
the POST will work withOUT any symbol import?

Reply:
No, `numbers` import required for `POST`. 
Fixed `numbers` import.